### PR TITLE
DOCS-1319: Filter and add correct link for input controller API missing methods

### PR DESCRIPTION
**Pull request by sguequierre**
For two missing methods: 
- `trigger_event` has a note not to be documented without full implementation (raises not implemented error) and `reset_channel` I _think_ is random helper function that shouldn't be documented, no equivalent in Go SDK. 

Paging @cheukt to confirm: is it best not to document [`trigger_event`](https://python.viam.dev/autoapi/viam/components/input/client/index.html#viam.components.input.client.ControllerClient.trigger_event) and [`reset_channel`](https://python.viam.dev/autoapi/viam/components/input/client/index.html#viam.components.input.client.ControllerClient.reset_channel) at the moment, or are they ready to be documented?

### CODE REVIEW

**Comment by Otterverse** [comment]
> For two missing methods:
> 
> * `trigger_event` has a note not to be documented without full implementation (raises not implemented error) and `reset_channel` I _think_ is random helper function that shouldn't be documented, no equivalent in Go SDK.
> 
> Paging @Otterverse to confirm: is it best not to document [`trigger_event`](https://python.viam.dev/autoapi/viam/components/input/client/index.html#viam.components.input.client.ControllerClient.trigger_event) and [`reset_channel`](https://python.viam.dev/autoapi/viam/components/input/client/index.html#viam.components.input.client.ControllerClient.reset_channel) at the moment, or are they ready to be documented?

I'm probably not the one to ask, as I don't know why that note is there or what reset_channel does. Definitely check with the python-sdk folks.

**Comment by sguequierre** [comment]
> > For two missing methods:
> > 
> > * `trigger_event` has a note not to be documented without full implementation (raises not implemented error) and `reset_channel` I _think_ is random helper function that shouldn't be documented, no equivalent in Go SDK.
> > 
> > Paging @Otterverse to confirm: is it best not to document [`trigger_event`](https://python.viam.dev/autoapi/viam/components/input/client/index.html#viam.components.input.client.ControllerClient.trigger_event) and [`reset_channel`](https://python.viam.dev/autoapi/viam/components/input/client/index.html#viam.components.input.client.ControllerClient.reset_channel) at the moment, or are they ready to be documented?
> 
> I'm probably not the one to ask, as I don't know why that note is there or what reset_channel does. Definitely check with the python-sdk folks.

Ok thank you! Bringing Cheuk in on this instead, I'll drop you from review

**Comment by cheukt** [comment]
reset_channel shouldn't be documented, I don't see the note to not document `trigger_event`. The NotSupportedError only shows up if the person implementing an input controller model doesn't override the method, which I think is similar to how go implements trigger events

**Comment by sguequierre** [comment]
> reset_channel shouldn't be documented, I don't see the note to not document `trigger_event`. The NotSupportedError only shows up if the person implementing an input controller model doesn't override the method, which I think is similar to how go implements trigger events

Ok! Sorry for the confusion, it was a docs comment on that file so I don't know how you would have seen it. It said "should not be documented until support is implemented for all input controller models". I am good with documenting unless it's still not ready. What do you think? Uncommented and added for now @cheukt

**Comment by cheukt** [comment]
@sguequierre I think we should document

File: .github/workflows/check_python_methods.py [hunk 1]
```diff
@@ -219,7 +219,9 @@ def parse(type, names):
                         break
 
             if not found and id != "viam.components.board.client.DigitalInterruptClient.add_callback" \
-            and id != "viam.components.board.client.DigitalInterruptClient.add_post_processor":
+            and id != "viam.components.board.client.DigitalInterruptClient.add_post_processor" \
+            and id != "viam.components.input.client.ControllerClient.reset_channel" \
+            and id != "viam.components.input.client.ControllerClient.trigger_event":
```
Comment (cheukt) [inline .github/workflows/check_python_methods.py:224]: should this be removed then?

**Review by npentrel — APPROVED** [review]

File: .github/workflows/check_python_methods.py (hunk 1 above)
Comment (sguequierre) [reply .github/workflows/check_python_methods.py]: Done, thank you!

**Pull request merged**